uint32 n_object
uint32 max_attempts
string data
---
bool success
string message